test/reporter.spec.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-unused-expressions */
 /* eslint-env mocha */
 var expect = require('chai').expect
-var gutil = require('gulp-util')
+var Vinyl = require('vinyl')
```

Comment:
Please move this line down to keep it in alphabetical order.

Reply:
Will fix.